backend/src/modules/match-module/core/domain/Match.ts

```diff
@@ -72,6 +72,9 @@ export function endMatch(
   if (!state.isParticipating(command.winnerId)) {
     throw new Error('One of the participating teams must be a winner.');
   }
+  if (state.winner) {
+    throw new Error('Cannot end match that has already ended.');
```

Comment:
Nie wiem czy to jest po angielskiemu. Ale to chyba całej apce by się przydało takie review :)

Reply:
też nie wiem czy Anglik by tak powiedział :P